packages/react/src/components/Layer/Layer.notification.ts

```diff
@@ -112,6 +113,32 @@ export function unregisterLayerHost(hostId: string, layerHost: ILayerHost): void
   }
 }
 
+/**
+ * When no default layer host is provided, this function is executed to create the default host.
+ */
+export function createDefaultLayerHost(doc: Document): Node | null {
+  const host = doc.createElement('div');
+  host.setAttribute('id', 'fluent-default-layer-host');
```

Comment:
nit: Replace `'fluent-default-layer-host'` with `defaultHostId`.

Reply:
Oh oops! meant to do that!

Reply:
done.